client/components/theme/style.scss

```diff
@@ -135,53 +142,46 @@ $theme-info-height: 54px;
 	font-weight: 600;
 }
 
-.theme__active-focus {
-	opacity: 0.0;
+.theme__thumbnail {
+	opacity: 1;
 	transition: all 200ms ease-in-out;
 
 	&:hover,
 	&:focus {
-		opacity: 1.0;
+		opacity: 0.9;
 
-		span {
-			animation: theme__active-focus-label 150ms ease-in-out;
+		.theme__thumbnail-label {
+			opacity: 1;
+			animation: theme__thumbnail-label 150ms ease-in-out;
 		}
 	}
 
-	.is-actionable & {
-		position: absolute;
+	.theme__thumbnail-label {
+		opacity: 0;
+		pointer-events: none;
+
 		z-index: z-index( 'root', '.is-actionable .theme__active-focus' );
```

Comment:
If the `.theme__active-focus` class is going away, please change also the z-index constant in `shared/functions/_z-index.scss`.